DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -250,6 +274,14 @@ function PreFirstEntryMapFixes()
         ft.bSetFlag=False;
         ft.bTrigger=True;
 
+        foreach AllActors(class'#var(prefix)ScriptedPawn',p){
+            if(p.BindName=="Supervisor01"){
+                p.FamiliarName="Mr. Hundley"; //It's spelled this way everywhere else
+                GiveItem(p,class'#var(prefix)Credits');  //He asks for 2000 credits to get into the labs
+                GiveItem(p,class'#var(prefix)Credits');  //He's probably getting a bunch of cash from other people too.
```

Comment:
is it normal to get them 2 stacks of credits? alternatively we could grab the return value from GiveItem and set the number of credits in it

Reply:
The Lucky Money door girl is the only person in the game with something other than 100 credits, and she just has 9 stacks on her.